core/controllers/access_validators.py

```diff
@@ -143,3 +146,44 @@ class ReleaseCoordinatorAccessValidationHandler(base.BaseHandler):
     def get(self) -> None:
         """Handles GET requests."""
         pass
+
+
+class ViewLearnerGroupPageAccessValidationHandler(base.BaseHandler):
+    """Validates access to view learner group page."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    # Type[str, Any] is used to match the type defined for this attribute in
+    # its parent class `base.BaseHandler`.
+    URL_PATH_ARGS_SCHEMAS: Dict[str, Any] = {
```

Comment:
@sahiljoster32 Is this the correct appraoch?

Reply:
In general can you please check the typing here?

Reply:
@sahiljoster32 asked me to remove the typings from `URL_PATH_ARGS_SCHEMAS` and have Dict[str, str] for `HANDLER_ARGS_SCHEMAS` for now. Have made the updates, thanks!

Reply:
Did a full pass on .py files!